prime-dotnet-webapi/Models/Agreements/AgreementType.cs

```diff
@@ -20,7 +20,7 @@ public enum AgreementType
         [AgreementGroups(AgreementGroup.Organization)]
         CommunityPharmacyOrgAgreement = 5,
 
-        [AgreementGroups(AgreementGroup.Enrollee)]
+        [AgreementGroups(AgreementGroup.Enrollee, AgreementGroup.OnBehalfOf)]
```

Comment:
Good use of the groups 👍

Reply:
Thanks for introducing this @james-hollinger ... meta-data for enums!